docs-config.yaml

```diff
@@ -0,0 +1,42 @@
+port: 3000
+host: "localhost"
+uploads:
+  driver: "local"
+  local:
+    path: "./uploads"
+password: secretpassword
+favicon: "public/favicon.png"
+frontend:
+  title: Editor.js
+  startPage: ""
+  description: Free block-style editor with a universal JSON output
+  menu:
+    - title: Documentation
+      uri: "/base-concepts"
+    - title: Awesome Plugins
+      uri: https://github.com/editor-js/awesome-editorjs
+    - title: Digest
+      uri: https://digest.editorjs.io/
+    - title: GitHub
+      uri: https://github.com/codex-team/editor.js
+    - title: Support Editor.js
+      uri: https://opencollective.com/editorjs
+  misprintsChatId: ""
+  yandexMetrikaId: ""
+  carbon:
+    serve: ""
+    placement: ""
+
+auth:
```

Comment:
why `password` is not under this section?

Reply:
you are right, it must be under auth section

can you move it pls